scripts/travis/permissions-workaround.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$(whoami)" != 'root' ]; then
+    echo 'Please run with sudo'
+    exit 2
+fi
+
+UBUNTU_VERSION=`lsb_release -a 2>&1 | grep Codename | grep -v "LSB" | awk '{print $2}'`
+
+# Workaround for Travis on Ubuntu Xenial/Bionic so local unit and packs tests work
+# when executing them under user "stanley" (by default Travis checks out the
+# code and runs tests under a different system user).
+if [ "${UBUNTU_VERSION}" == "xenial" ] || [ "${UBUNTU_VERSION}" == "bionic" ]; then
+  echo "Applying workaround for stanley user permissions issue to /home/travis on Xenial/Bionic"
+  chmod -R a+rwX /home/travis
```

Comment:
Would it make more sense to change the ownership of some of the files in `/home/travis` to `stanley` instead of trying to `chmod` them to the appropriate settings?

And even if it still makes sense to use `chmod`, we still should not be setting all of the bits. We should be as selective as possible.